### DISCUSSION TITLE
New Github actions aren't displaying in actions tab

### DISCUSSION THREAD

**Discussion by igorpetuh** [community/community#26443] [Actions]
<p>I was trying to create a new action from templates for my repo today 29.04.2020. But new actions are not displayed. Last week everything worked pretty well. Also tried to reproduce it in another repositories and got the same problem</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/8/83839bbd936520ab90972ec297560b9e8a53ba3c.png" data-download-href="/uploads/short-url/iLqw87LFHwgwYUxNHiPBjuUq4gs.png?dl=1" title="Screen Shot 2020-04-29 at 15.24.05.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/11535769/181097843-e8b4b5a3-4d7d-44d5-96ed-1b844400322f.png" alt="Screen Shot 2020-04-29 at 15.24.05.png" data-base62-sha1="iLqw87LFHwgwYUxNHiPBjuUq4gs" width="407" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/8/83839bbd936520ab90972ec297560b9e8a53ba3c_2_407x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/8/83839bbd936520ab90972ec297560b9e8a53ba3c_2_610x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/8/83839bbd936520ab90972ec297560b9e8a53ba3c_2_814x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/8/83839bbd936520ab90972ec297560b9e8a53ba3c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-04-29 at 15.24.05.png</span><span class="informations">830×1018 27.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div> <div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/a/a8f1ef664bb83fc0e4683235cade65287df7cd2a.png" data-download-href="/uploads/short-url/o6yyDH7IdijJplqRCKhlogJSpOa.png?dl=1" title="Screen Shot 2020-04-29 at 15.25.59.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/11535769/181097844-05fc746b-9986-478a-9645-2a8b234dcca1.png" alt="Screen Shot 2020-04-29 at 15.25.59.png" data-base62-sha1="o6yyDH7IdijJplqRCKhlogJSpOa" width="690" height="478" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/a/a8f1ef664bb83fc0e4683235cade65287df7cd2a_2_690x478.png, https://aws1.discourse-cdn.com/github/optimized/2X/a/a8f1ef664bb83fc0e4683235cade65287df7cd2a_2_1035x717.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/a/a8f1ef664bb83fc0e4683235cade65287df7cd2a_2_1380x956.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/a/a8f1ef664bb83fc0e4683235cade65287df7cd2a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-04-29 at 15.25.59.png</span><span class="informations">1714×1188 214 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/igorpetuh">@igorpetuh</a> ,</p>
<p>The created workflow will not be listed on the <strong>Actions</strong> tab before its first run is triggered. So, please make sure the new created workflow has been triggered at least once, if you want to view it on the <strong>Actions</strong> tab.</p>

**Comment by igorpetuh** [comment]
<p><a class="mention" href="/brightran">@brightran</a> </p>
<p>Looks like you’re right. But it’s not a really obvious behavior. And i’m not sure that it’s even documented.</p>
